odoo/models.py

```diff
@@ -3109,7 +3116,18 @@ def _read_format(self, fnames, load='_classic_read'):
         data = [(record, {'id': record._ids[0]}) for record in self]
         use_name_get = (load == '_classic_read')
         for name in fnames:
-            convert = self._fields[name].convert_to_read
+            field = self._fields[name]
+            if field.type == 'properties':
+                values_list = [record[name] for record, _ in data]
+                records = self.browse([record.id for record, _ in data])
```

Comment:
you recreate `self`, no ?

Reply:
indeed, good catch :) but with the new MissingError change this is still needed